app/src/lib/app-theme.ts

```diff
@@ -34,6 +34,9 @@ export const appTheme = extendTheme({
       neutral: "#D7D8FA",
       overlay: "#E6E7FF",
     },
+    divider: {
+      grey03: "#F0F0F0",
```

Comment:
Weird name for this. Should it maybe just be `divider.neutral` or something like that?